modules/table/src/main/java/org/apache/ignite/internal/table/distributed/command/PartitionCommand.java

```diff
@@ -38,5 +41,15 @@ public interface PartitionCommand extends SafeTimePropagatingCommand, CatalogVer
      * Returns version that the Catalog must have locally for the node to be allowed to accept this command via replication.
      */
     @Override
+    @Transient
+    @WithSetter
     int requiredCatalogVersion();
+
+    /**
+     * Setter for {@link #requiredCatalogVersion()}. Called by the creator or the {@link PartitionCommandsMarshaller} while deserializing.
+     */
+    @Override
+    default void requiredCatalogVersion(int version) {
```

Comment:
Little strange

Reply:
What exactly is strange? The fact that javadoc differs from the parent class?

Reply:
Yep